dist_run.py

```diff
@@ -125,18 +125,39 @@ def _load_model_weights(
         distribution (str): The distribution name, e.g. "meta-llama/Meta-Llama-3-8B-Instruct".
         device (torch.device): The device to load the weights onto.
         model_config (ModelArgs): The model config.
-        chpt_from (str): The checkpoint format to load the weights from, e.g. "torchchat" or "hf".
+        chpt_from (str): The checkpoint format to load the weights from.
```

Comment:
as in prev comments, pls switch to 'chkpt' for abbreviating checkpoint.  It reads as 'chapter_from' when using 'chpt_from'.